### DISCUSSION TITLE
How hard would it to add support for multiple Lightpack devices back?

### DISCUSSION THREAD

**Discussion by pdecat** [hyperion-project/hyperion.ng#1482] [Q&A] [answered]
Hi,

I have been using Hyperion.ng with one Lightpack device happily for the last few years, but I've finally decided to replace my 55" TV by a 77" one, so a single Lightpack device is a bit short.

I have a second spare one as I purchased a 2x pack during the Lightpack Kickstarter campaign, but it is not detected since [support for multiple Lightpacks was removed](https://github.com/hyperion-project/hyperion.ng/pull/1049) to [simplify things and fix an issue](https://github.com/hyperion-project/hyperion.ng/issues/1015#issuecomment-699123730).

How hard do you think would it be to re-add support for multiple Lightpack devices?

**Comment by Lord-Grey** [comment, marked as answer]
Hi
Just add a 2nd instance in Hyperion and configure there the 2nd lightpack as the device for the instance.
That is also the way proposed when the multi-device via one config support was removed.

**Reply by pdecat** [reply]
Hi @Lord-Grey,
thanks for the quick reply! I never realized this was so easy to setup that way :facepalm: 
This is awesome, thanks again :)
